assignment1.txt

```diff
@@ -0,0 +1,5 @@
+1. If the code has been executed in the way it was meant to, the function return 0; tells the OS so. If the return value is 1, it implies there was a problem in the execution of the program 
+2. x is a floating point number, which means it has a decimal value. The use of the command %d instructs the program to output an integer value, which x is not
```

Comment:
Correct! You're missing a couple more errors - check and see if you can find them!